README.md

```diff
@@ -0,0 +1,61 @@
+# Description
+
+Simple pure python utility for sniffing HTTP/HTTPS decrypted traffic recorded by one of Apple's not-so-well documented
+APIs.
+
```

Comment:
- explain more about the workflow
- we sniff the RAW decrypted data, then simulate a "semi-equivalent" TCP stream to be able to view diagnose this with other tools...